release_notes.md

```diff
@@ -1,3 +1,82 @@
+***********************************
+# Release Notes for OpenMDAO 3.35.0
+
+Sept 30, 2024
+
+OpenMDAO 3.35.0 includes a significant change in the way outputs from a particular OpenMDAO execution are handled. These changes are summarized in [POEM 097](https://github.com/OpenMDAO/POEMs/pull/198). Previously, outputs from a run were stored in various locations, including the current working directory, and a reports directory. Derivative coloring files were stored in a coloring_dir folder in the current working directory, and all of these could be overridden by running different OpenMDAO scripts in the same directory on the user's computer.
+
+- OpenMDAO now puts all output files and coloring files associated with a given execution in a folder named after the problem. One can control the name of this folder by setting a name for a given problem.
+- Subproblems are stored in folders under the parent problem.
+- The user may specify a coloring directory that contains "input" coloring files, to be loaded. The coloring algorithm will always save coloring files for the current problem in the problems output folder.
+- The top-level directory under which these run directories are placed is controlled with the `OPENMDAO_WORKDIR` environment variable, which defaults to the current working directory.
+
+** Note that these changes can cause a huge number of output directories to be placed into the users hard drive when running many OpenMDAO problems, such as when running tests **
+
+We recommend users run tests using temporary directories (see our @use_tempdirs decorator for tests).
+
+Additionally, we now provide an `openmdao.api.clean_outputs()` method as well as a command line form `openmdao clean` or `python -m openmdao clean`.  See the help for these commands for more information, but they'll make your life a bit less chaotic.
+
+In other highlights...
+
+- We now support of numpy 2.0.
+- A new implicit component, `InputResidsComp` has been added. This is probably the simplest possible use case for an implicit component, where the residuals are given as inputs.
+- We've added a new `setup_residuals` method in `ImplicitComponent`, which like `setup_partials`, delays the addition of residuals until the shape of inputs and outputs are known.
+- The OpenMDAO command-line tools are accessible via the more pythonic `python -m openmdao` command. We also added a `view_reports` subcommand to quickly bring up the reports associated with a given case.
```

Comment:
I think that this is not quite correct..  

`view_reports` was already a subcommand that opens a browser for all reports found under the current directory.

The new functionality is the `--view_reports` option to the bare `openmdao` command that will open a browser to view reports for a problem after running the script